alibi_detect/saving/saving.py

```diff
@@ -110,7 +117,9 @@ def _cleanup_filepath(orig_files: set, filepath: Path):
 
 
 # TODO - eventually this will become save_detector (once outlier and adversarial updated to save via config.toml)
-def _save_detector_config(detector: ConfigurableDetector, filepath: Union[str, os.PathLike]):
+def _save_detector_config(detector: ConfigurableDetector,
```

Comment:
do we not want to include StatefulDetector here in type hints?

Reply:
Good idea!